src/ilc-adapter-react.test.tsx

```diff
@@ -90,24 +92,29 @@ describe('ilc-adapter-react', () => {
         });
     });
 
-    it(`mounts and unmounts an APP, passing through the ILC props`, () => {
+    it(`mounts and unmounts an APP, passing through the ILC props`, async () => {
         const lifecycles = ilcAdapterReact<AppLifecycleFnProps>({
             rootComponent: (props: AppLifecycleFnProps) => <div>Hello world!</div>,
         });
 
-        return lifecycles
-            .bootstrap(appProps)
-            .then(() => lifecycles.mount(appProps))
-            .then(() => {
-                expect(root.querySelector('div')?.textContent).toEqual('Hello world!');
-                return lifecycles.unmount(appProps);
-            })
-            .then(() => {
-                expect(root.childElementCount).toEqual(0);
-            });
+        await lifecycles.bootstrap(appProps);
+
+        await lifecycles.mount(appProps);
+
+        await pause();
```

Comment:
pauses look strange, could you please provide some comment with small description

Reply:
can you see the implementation of this helper?
it is just requestAnimationFrame under the hood because rendering is not sync operation

Reply:
I see, but previously it was unnecessary and now necessary, so it can confuse

Reply:
that because react render mechanism was changed